Slider/tests/Slider-specs.js

```diff
@@ -13,6 +13,35 @@ const rightKeyDown = keyDown(39);
 const upKeyDown = keyDown(38);
 const downKeyDown = keyDown(40);
 
+const getElementClientCenter = (element) => {
+	const {left, top, width, height} = element.getBoundingClientRect();
+	return {x: left + width / 2, y: top + height / 2};
+};
+
+const sleep = ms =>
```

Comment:
I'm not sure this kind of code works or not.
maybe we replaced all setTimeout to "advanceTimersByTime".
in https://github.com/enactjs/sandstone/pull/1216/files

 @sjro  What do you think?